src/ProgressOnderwijsUtils/Data/SqlServerUtils.cs

```diff
@@ -34,7 +34,8 @@ static bool IsSpidAlreadyDeadException(Exception exception)
 
     public static string PrettifySqlExpression(string sql)
     {
-        var withoutPointlessParens = Regex.Replace(sql, @"\((\w+)\)", m => m.Value[1..^1]);
+        var alomstPretty = PrettifySqlExpressionLeaveParens(sql);
```

Comment:
I can't make anything of the variablename. Maybe inline?

Reply:
I have now renamed the variable. If I inline it, then R# wants me to convert it to an expression body, and afterwards the functionality can only be deducted from the code itself (and regular expressions are not obvious to everyone).